bundles/aliyun/build.gradle.kts

```diff
@@ -39,6 +39,10 @@ dependencies {
   implementation(project(":catalogs:hadoop-common")) {
     exclude("*")
   }
+  implementation(project(":clients:client-java-runtime", configuration = "shadow"))
```

Comment:
Again, why this should be `implementation`, AFAIK, the bundles are typically used with gvfs, we already included this client runtime in gvfs, what's the reason to do this here?

Reply:
Let me think a bit.